bioio_ome_zarr/reader.py

```diff
@@ -96,10 +92,33 @@ def _is_supported_image(
 
         try:
             store = fs.get_mapper(path)
-            zarr.open_group(store=store, mode="r")
+            group = zarr.open_group(store=store, mode="r")
+            attrs = group.attrs.asdict()
+
+            # Check for transitional metadata key
+            if ("bioformats2raw.layout" in attrs) or (
+                isinstance(attrs.get("ome"), dict)
+                and "bioformats2raw.layout" in attrs["ome"]
+            ):
+                raise exceptions.UnsupportedFileFormatError(
+                    Reader.__name__,
+                    path,
+                    (
+                        "Detected transitional layout metadata key "
+                        "'bioformats2raw.layout'. This layout describes multiple "
+                        "image series, not a single image. BioIO does *not* support "
+                        "reading stores using this format."
```

Comment:
Maybe the output could suggest something like "Please provide the path to a single ome-zarr image"